sqlitecluster/SQLiteNode.cpp

```diff
@@ -520,11 +521,14 @@ bool SQLiteNode::update() {
                         freshestPeer = peer;
 
                     // See if it's the highest priority
-                    if (!highestPriorityPeer || peer->calc("Priority") > highestPriorityPeer->calc("Priority"))
+                    const string& peerState = (*peer)["State"];
+                    int peerEffectivePriority = SIEquals(peerState, "SYNCHRONIZING") ? 0 : peer->calc("Priority"); 
```

Comment:
i think this needs to be 1 instead of 0 or it will cause some oddness with permaslave flows.

Reply:
What oddness?

Reply:
I'm not sure I can point to any -- just that there are places where 0 is used to denote what should be a permaslave and what should not be.